### ISSUE TITLE
Scrolling timeline

### ISSUE THREAD

**Issue by eliaswood** [timhaley94/election_simulator#2]
I think it would be really cool if the timeline shifted as the parties approached the end of the available space. When it first shifts, a scroll bar could appear below to then let you move back and forth to see the trend.

**Comment by timhaley94** [comment]
Yes that would be awesome! That relates to what I have in TODO.md of "removing the iteration cap". Basically, just letting the simulation go forever until you stop it. In that case, it would be needed for the graph to be scrollable.
